### DISCUSSION TITLE
Object column

### DISCUSSION THREAD

**Discussion by jponce-meyler** [azimuttapp/azimutt#250] [Q&A]
There is an example for MongoDB with complex objects as a columns. Is it possible to do the same with the AML syntax?

**Comment by loicknuchel** [comment]
Hi @jponce-meyler 

I missed alerts so just saw your message now, sorry for the delay.
To answer your question it's not, not yet. It's planned to improve AML to handle that and other things but not done yet...
I created [an issue](https://github.com/azimuttapp/azimutt/issues/267) if you want to follow progress and discuss other evolutions ;)

Cheers!

**Comment by jponce-meyler** [comment]
Don't be sorry, the tool is awesome. We are a small team but we are planning to grow up and we will for sure purchase the membership.
Anyways I suggest to you to put some json of the examples to have a clear idea how to do it. Was a real pain for us understand how to make the relationships in case of nested objects that has references to other "tables/collections".
Thank you very much for you job!

**Comment by loicknuchel** [comment]
Thank you so much for your kind words, it's very valuable to me <3
Would be happy to count your team as customer ^^ Don't hesitate if you want to do a call to discuss your use case and how it fit with Azimutt, it's always insightful for me to learn how users use Azimutt and their specific needs.

For the JSON example, I have a JSON Schema in the project creation but it's indeed missing from the source creation modal (and it's a bit different ^^).
Will add it.
Do you see elsewhere it would be obvious to add?

If you struggle at anything don't hesitate to reach out, here, on Slack or by email ;)

Cheers!

**Comment by loicknuchel** [comment]
Added the JSON Schema next to the JSON input so it should be easier to spot: https://github.com/azimuttapp/azimutt/commit/622af44836c7730e6338d8be12ea67dd958d0648
It's deployed now.

Tell me if you think I can do more on this.
Thanks
